lib/components/user/monitored-trip/saved-trip-screen.js

```diff
@@ -27,13 +28,13 @@ const validationSchemaShape = {
   isActive: yup.boolean(),
   leadTimeInMinutes: yup.number().positive().integer(),
   tripName: yup.string()
-    .required('Please enter a trip name')
+    .required('components.SavedTripScreen.tripNameRequired')
```

Comment:
Please do not pass message IDs like this to yup validation schemas. This results in message IDs not being hard-coded within a `intl.format` call or `FormattedMessage` component and is therefore hard to automatically keep track of which IDs are in the code.

Instead, please omit the message from the yup validation schema:

```js
tripName: yup.string().required()
```